src/client/src/components/Details/index.tsx

```diff
@@ -51,7 +51,7 @@ const Details = ({
     </React.Fragment>
   );
   const keyDownHandler = (event: any) => {
```

Comment:
change all any to React.KeyboardEvent happens in multiple places

Reply:
done :)